docs/swagger/swagger.json

```diff
@@ -96,6 +96,37 @@
         }
       }
     },
+    "/user/forgotpassword/{email}" : {
+      "post" : {
+        "tags" : [ "user" ],
+        "summary" : "sends a reset password email",
+        "description" : "The forgot password endpoint.",
+        "parameters" : [ {
+          "in" : "path",
+          "name" : "email",
+          "schema" : {
+            "type" : "string",
+            "format" : "email"
+          },
+          "required" : true,
+          "description" : "email of the user"
+        }],
+        "responses" : {
+          "200" : {
+            "description" : "Reset password email sent"
+          },
+          "404" : {
```

Comment:
We don't want to give any indication of if an email does or does not exist in the system. We should remove this and return the same `200` as an appropriate email.